src/main/java/io/github/apace100/origins/util/Comparison.java

```diff
@@ -9,7 +9,8 @@
     LESS_THAN("<", (a, b) -> a < b),
     GREATER_THAN(">", (a, b) -> a > b),
     LESS_THAN_OR_EQUAL("<=", (a, b) -> a <= b),
-    GREATER_THAN_OR_EQUAL(">=", (a, b) -> a >= b);
+    GREATER_THAN_OR_EQUAL(">=", (a, b) -> a >= b),
+    NOT_EQUAL("!=", (a, b) -> a != b);
```

Comment:
Not equal comparison should use `!a.equals(b)` because they are number objects, not primitives.